howto/README.md

```diff
@@ -0,0 +1,49 @@
+# How do I ...
+
```

Comment:
This may be premature optimization, but what about having a table of contents at the top? 

I imagine this list might grow quickly and be hard to see what's included in it